warrior/WarriorCore/Classes/testcase_utils_class.py

```diff
@@ -195,7 +195,7 @@ def p_note_level(self, txt, print_type="info", level=None, ptc=True):
         """Create Note at the provided level"""
         write_locn = self.get_write_locn(str(level).upper())
         print_util_types = ["-D-", "", "-I-", "-E-", "-W-",
-                            "\033[1;31m-E-\033[0m"]
+                            "\033[1;31m-E-\033[0m", "\x1b[1;33m-W-\x1b[0m"]
```

Comment:
@safarheen this change works. Just to maintain uniformity, can you add octal representation instead of hex.
"\033[1;33m"+ "-W-" + "\033[0m" instead of "\x1b[1;33m-W-\x1b[0m".

Python internally converts this from octal to hex.

Reply:
Addressed the comments.